woocommerce/wc-admin
# Move the shipping input and text 1px lower

Fixes #6394 

Move the shipping input and text 1px lower

### Screenshots
Before
<img width="923" alt="螢幕快照 2021-02-22 下午4 23 15" src="https://user-images.githubusercontent.com/56378160/108773779-dc4d5b00-752c-11eb-8bb7-96550013fdb9.png">

After
<img width="800" alt="after" src="https://user-images.githubusercontent.com/56378160/108773794-e1120f00-752c-11eb-9c7e-6546043d1cd1.png">


### Detailed test instructions:

-   Go to WooCommerce home.
-   Click on 'Set up shipping costs'.
-   See the letters "p" and "g" in the 'Shipping cost' render as expected.
